engine/apps/slack/alert_group_slack_service.py

```diff
@@ -35,22 +35,28 @@ def __init__(
             self._slack_client = SlackClient(slack_team_identity)
 
     def update_alert_group_slack_message(self, alert_group: "AlertGroup") -> None:
-        logger.info(f"Update message for alert_group {alert_group.pk}")
+        alert_group_pk = alert_group.pk
+
+        logger.info(f"Update message for alert_group {alert_group_pk}")
 
         try:
             self._slack_client.chat_update(
-                channel=alert_group.slack_message.channel.slack_id,
+                # TODO: once _channel_id has been fully migrated to channel, remove _channel_id
+                # see https://raintank-corp.slack.com/archives/C06K1MQ07GS/p173255546
+                # channel=alert_group.slack_message.channel.slack_id,
+                channel=alert_group.slack_message._channel_id,
```

Comment:
"fix" from https://github.com/grafana/oncall/pull/5292 (let's wait to read from `channel` until the data-migration mentioned in that PR has been fully finished)